airflow/api_connexion/openapi/v1.yaml

```diff
@@ -276,6 +276,34 @@ paths:
                   - $ref: '#/components/schemas/CollectionInfo'
         '401':
           $ref: '#/components/responses/Unauthenticated'
+    
+    post:
+      summary: Trigger a DAG Run
+      operationId: airflow.api_connexion.endpoints.dag_run_endpoint.post_dag_run
+      tags: [DAGRun]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/DAGRun'
+      responses:
+        '200':
+          description: Successful response.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DAGRun'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthenticated'
+        '409':
+          $ref: '#/components/responses/AlreadyExists'
+        '403':
+          $ref: '#/components/responses/PermissionDenied'
+        '404':
+          $ref: '#/components/responses/NotFound'
```

Comment:
I moved the `post request` from endpoint `/dags/{dag_id}/dagRuns/{dag_run_id}` to `/dags/{dag_id}/dagRuns` as having dag_run_id as a url parameter as well as in request.body seemed redundant.